### PR TITLE
[PLA-29] Extract marketing site

### PR DESCRIPTION
**Pull request by MythicalFish**
- Change root route to dashboard
- Removed routes
- Removed controller (preserving the RSS action)
- Removed views & application layout
- Renamed gsdk layout to application & made it global

### CODE REVIEW

File: app/controllers/coins_controller.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class CoinsController < ApplicationController
+  
+  # TODO: Respond with JSON for use on marketing site?
```
Comment (hanchang) [inline app/controllers/coins_controller.rb]: No need, marketing site will just be archived HTML/CSS

File: config/routes.rb [hunk 2]
```diff
@@ -2,14 +2,20 @@
   mount Blazer::Engine, at: "blazer"
 
   devise_scope :user do
+    # TODO: (possibly)
```
Comment (hanchang) [inline config/routes.rb]: Hm, I think you're right - feel free to refactor this appropriately, thanks!

Comment (MythicalFish) [reply config/routes.rb]: Had a go at this, but will leave it for now as it's confusing me a little. Also I think additional, separate controllers are needed for things like contributions. Might be worth making a task in Jira?

**Comment by hanchang** [comment]
@MythicalFish I took the liberty of running `rails webpacker:install` on this branch before I noticed your progress on https://github.com/coinfi/coinfi/pull/25; please lemme know if this wasn't part of the plan!

That said though, the Heroku review app autodeploy is now working so maybe it's fine?

**Comment by MythicalFish** [comment]
@hanchang Just adding the webpacker gem solved the missing dependency issue, but indeed the app wouldn't run without running `webpacker:install`, so I then tried it out with the webpacker branch I made previously. I'll remove webpacker from this branch, and the auto-deploys will be fixed when the other branch gets merged. :) Hope that makes sense

**Comment by MythicalFish** [comment]
@hanchang Ok I think I'm done here :)

**Comment by hanchang** [comment]
@MythicalFish Apologies for the inconvenience but after merging https://github.com/coinfi/coinfi/pull/22 seems like there's a bunch of merge conflicts now that need your help to resolve - thanks!

**Comment by MythicalFish** [comment]
@hanchang Done!

**Comment by hanchang** [comment]
Looks great! 💯

**Pull request merged**